### ISSUE TITLE
How to remove CTR after Fx 57+ is installed?

### ISSUE THREAD

**Issue by Tetecarre** [Aris-t2/ClassicThemeRestorer#388]
I have several lines in about:config that are from classicthemerestorer. I guess they were placed there by sync.

**Comment by Aris-t2** [comment]
Right-click on them an select "Reset". They will be gone after a restart.
